src/commands/info.go

```diff
@@ -0,0 +1,173 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/bwmarrin/discordgo"
+	"log"
+	"runtime"
+	"sandwich-delivery/src/database"
+	"sandwich-delivery/src/models"
+	"strconv"
+)
+
+type InfoCommand struct{}
+
+func (c InfoCommand) getName() string {
+	return "info"
+}
+
+func (c InfoCommand) getCommandData() *discordgo.ApplicationCommand {
+	return &discordgo.ApplicationCommand{Name: c.getName(),
+		Description: "Lookup info of the bot or a user.",
+		Options: []*discordgo.ApplicationCommandOption{
+			{
+				Name:        "user",
+				Description: "Lookup an user.",
+				Options: []*discordgo.ApplicationCommandOption{
+					{
+						Type:        discordgo.ApplicationCommandOptionUser,
+						Name:        "user",
+						Description: "The user to lookup.",
+						Required:    true,
+					},
+				},
+				Type: discordgo.ApplicationCommandOptionSubCommand,
+			},
+			{
+				Name:        "bot",
+				Description: "Lookup Bot Data",
+				Type:        discordgo.ApplicationCommandOptionSubCommand,
+			},
+		},
+	}
+}
+
+func (c InfoCommand) DMsAllowed() bool {
+	return true
+}
+
+func (c InfoCommand) registerGuild() string {
+	return ""
```

Comment:
admin commands should not be global

Reply:
This actually was not supposed to be an admin command, it is fixed now.